lib/recognize-stream.ts

```diff
@@ -111,6 +112,7 @@ class RecognizeStream extends Duplex {
    * @param {Number} [options.X-Watson-Learning-Opt-Out=false] - set to true to opt-out of allowing Watson to use this request to improve it's services
    * @param {Boolean} [options.smart_formatting=false] - formats numeric values such as dates, times, currency, etc.
    * @param {String} [options.customization_id] - Customization ID
+   * @param {String} [options.acoustic_customization_id] - Acoustic customization ID
```

Comment:
https://www.ibm.com/watson/developercloud/speech-to-text/api/v1/curl.html?curl#websocket_methods